src/Whoops/RunInterface.php

```diff
@@ -25,6 +25,15 @@
      */
     public function pushHandler($handler);
 
+    /**
+     * Prepends a handler to the begining of the stack
+     *
+     * @throws InvalidArgumentException  If argument is not callable or instance of HandlerInterface
+     * @param  Callable|HandlerInterface $handler
+     * @return Run
+     */
+    public function prependHandler($handler);
```

Comment:
Changing the interface is a BC break. We need to check whether this is a problem..

Reply:
Is it OK to add new function but not update the interface?

Reply:
@denis-sokolov @staabm Is it OK?